### PR TITLE
streetviewpanorama status

### PR DESCRIPTION
**Pull request by nmccready**
Adds the ability to track the status of the StreetViewPanorama image. This way we can take action on it if there image is something other than "OK" .

### CODE REVIEW

**Comment by nmccready** [comment]
@zacronos

File: example/assets/scripts/controllers/templates/app-window-ctrl.js [hunk 1]
```diff
@@ -4,4 +4,10 @@ angular.module('app')
     $scope.getParam = function(){
```
Comment (nmccready) [inline example/assets/scripts/controllers/templates/app-window-ctrl.js]: example

File: src/coffee/directives/street-view-panorama.coffee [hunk 2]
```diff
@@ -22,6 +22,7 @@ angular.module('uiGmapgoogle-maps')
       options: '=?'
       control: '=?'
```
Comment (nmccready) [inline src/coffee/directives/street-view-panorama.coffee]: only changes

**Comment by zacronos** [comment]
LGTM

**Pull request merged**